recipe/meta.yaml

```diff
@@ -90,23 +93,30 @@ outputs:
         - cuda-version {{ cuda_major }}.*        # [cuda_compiler_version != "None"]
       run:
         - python
+        # requirements/pip/nvmath-python.txt
+        - cuda-bindings
+        - cuda-core 0.2.*
         - numpy >=1.24,<3
         - pywin32                         # [win64]
+        # requirements/pip/nvmath-python-cu12.txt
         - cupy                            # [cuda_compiler_version != "None"]  # TODO: remove me in the next release
```

Comment:
Can we remove it now?

Reply:
No. It's required for host-device transfers when the user chooses to use numpy as the input tensor.

Reply:
Ah ok, yes the copy routine is still not ready.